test/vm-prep

```
@@ -79,6 +79,12 @@ EOF
         echo "$rule" >> ${qemu_config_dir}/bridge.conf
       fi
   done
+
+    # We really want nested virtualization
+    if [ ! -f /etc/modprobe.d/kvm-intel.conf ]; then
```

It's not a constraint. Failures are ignored.

But i'll add support for AMD.